src/main/java/io/getunleash/DefaultUnleash.java

```diff
@@ -190,6 +190,7 @@ public Variant getVariant(String toggleName, Variant defaultValue) {
         return getVariant(toggleName, contextProvider.getContext(), defaultValue);
     }
 
+    @Override
     public Optional<FeatureToggle> getFeatureToggleDefinition(String toggleName) {
         return ofNullable(featureRepository.getToggle(toggleName));
     }
```

Comment:
This seems to me better placed in the MoreOperations interface (like `getFeatureToggleNames` - notice how the one in Unleash interface was deprecated) - You could then access it with `more().getFeatureToggleDefinition('some-feature')`

Reply:
Hey, thanks. that sounds cool. 

What implementation would you expect in `FakeUnleash.FakeMore`?
should we just throw a UnsupportedOperation exception for now or do we need to change this to return a dummy feature toggle?

Reply:
Thanks @andreas-unleash. I have now moved the method to the `MoreOperations` interface and changed `features` to be a map of `FeatureToggle` rather than a map of booleans.

Reply:
Thanks @Chaosmi ! Approved